plugins/catalog/src/components/CatalogTable/CatalogTable.tsx

```diff
@@ -64,6 +63,26 @@ const YellowStar = withStyles({
   },
 })(Star);
 
+/**
+ * MUI v5 removes `srOnly` variant from `Typography` component
+ *
+ * Utility copied from https://github.com/mui/material-ui/blob/master/packages/mui-utils/src/visuallyHidden.ts
+ * as `srOnly` variant was only used in this file.
+ */
+const useStyles = makeStyles({
+  visuallyHidden: {
```

Comment:
Tbh I think it would be completely fine to depend on `@mui/utils` directly and import from there instead. We actually already have it installed as a transitive dependency in the repo.

Reply:
yeah - not sure. Felt a bit uneccessary as this is really the only place `srOnly` is used & I'm not sure if there would be another place this util would be used. Should I change it for this PR or can we just go with this?

Reply:
I think it's best to introduce `@mui/utils` already tbh, shouldn't hurt much and we avoid the duplication of this logic.